### PR TITLE
Update testing section of the README

### PR DESCRIPTION
**Pull request by laurenyu**
need to reflect changes from #41

also added some general testing fixes - notably switched back to 'aws-id' to match PyTorch/Chainer (not sure why I switched to match TF in the first place).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -193,7 +192,7 @@ If you want to run local integration tests, then use:
 
 ::
 
-    # Required arguments for integration tests are found in test/integ/conftest.py
+    # Required arguments for integration tests are found in test/conftest.py
 
     pytest test/integ --docker-base-name <your_docker_image> \
```
Comment (icywang86rui) [inline README.rst]: should this be "pytest test/integ/local"

Comment (icywang86rui) [reply README.rst]: same on line 206. github does not let me comment there -__-

Comment (laurenyu) [reply README.rst]: good catch; fixed

File: README.rst [hunk 2]
```diff
@@ -193,72 +192,70 @@ If you want to run local integration tests, then use:
...
 
     # Example
-    pytest test/functional --aws-id 12345678910 \
-                           --docker-base-name preprod-mxnet \
-                           --instance-type ml.m4.xlarge \
-                           --tag 1.0
+    pytest test/integration/sagemaker --aws-id 12345678910 \
+                                      --docker-base-name preprod-mxnet \
+                                      --instance-type ml.m4.xlarge \
+                                      --tag 1.0
```
Comment (icywang86rui) [inline README.rst]: Let's be consistent with our example and do something like '0.12.1-gpu-py3' here

**Pull request merged**